migrate/v0_13/migrate.go

```diff
@@ -108,6 +110,106 @@ func MigrateAuth(genesisState auth.GenesisState) auth.GenesisState {
 	return genesisState
 }
 
+// Committee migrates from a v0.11 (or v0.12) committee genesis state to a v0.13 committee genesis stat
+func Committee(genesisState v0_11committee.GenesisState) v0_13committee.GenesisState {
+	committees := []v0_13committee.Committee{}
+	votes := []v0_13committee.Vote{}
+	proposals := []v0_13committee.Proposal{}
+
+	var newStabilityCommittee v0_13committee.Committee
+	var newSafetyCommittee v0_13committee.Committee
+
+	for _, com := range genesisState.Committees {
+		if com.ID == 1 {
+			newStabilityCommittee.Description = com.Description
+			newStabilityCommittee.ID = com.ID
+			newStabilityCommittee.Members = com.Members
+			newStabilityCommittee.VoteThreshold = com.VoteThreshold
+			newStabilityCommittee.ProposalDuration = com.ProposalDuration
+			var newStabilityCommitteePermissions []v0_13committee.Permission
+			var newStabilitySubParamPermissions v0_13committee.SubParamChangePermission
+
+			for _, perm := range com.Permissions {
+				subPerm, ok := perm.(v0_11committee.SubParamChangePermission)
+				if ok {
+					// update AllowedParams
+					var newAllowedParams v0_13committee.AllowedParams
+					for _, ap := range subPerm.AllowedParams {
+						newAP := v0_13committee.AllowedParam(ap)
+						newAllowedParams = append(newAllowedParams, newAP)
+					}
+					newStabilitySubParamPermissions.AllowedParams = newAllowedParams
+
+					// update AllowedCollateralParams
+					var newCollateralParams v0_13committee.AllowedCollateralParams
+					for _, cp := range subPerm.AllowedCollateralParams {
+						newCP := v0_13committee.AllowedCollateralParam(cp)
```

Comment:
Looks like go doesn't like the type cast to AllowedCollateralParam. Circleci doesn't run the migrate tests.
```suggestion
						newCP := v0_13committee.NewAllowedCollateralParam(
							cp.Type,
							cp.Denom,
							cp.LiquidationRatio,
							cp.DebtLimit,
							cp.StabilityFee,
							cp.AuctionSize,
							cp.LiquidationPenalty,
							cp.Prefix,
							cp.SpotMarketID,
							cp.LiquidationMarketID,
							cp.ConversionFactor,
							true,
							true,
						)
```